fixtures/build.js

```diff
@@ -22,28 +20,32 @@ window.EXAMPLE_BUILD = {
       }
     },
     "strategy": {
-      "type": "STI",
-      "stiStrategy": {
-        "builderImage": "openshift/ruby-20-centos7:latest",
-        "image": "openshift/ruby-20-centos7:latest"
+      "type": "Source",
+      "sourceStrategy": {
+        "from": {
+          "kind": "DockerImage",
```

Comment:
From the API doc:
```
From is reference to an ImageStream, ImageStreamTag, or ImageStreamImage
```
@bparees can you confirm those are the only valid kinds currently? Do we have examples of all the valid formats for the `name` value for each of those kinds?